FileProcessor/src/main/java/com/group7/FileProcessor/FileProcessorMain.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ */
+
+package com.group7.FileProcessor;
+
+import com.laserfiche.api.client.model.AccessKey;
+import com.laserfiche.repository.api.RepositoryApiClient;
+import com.laserfiche.repository.api.RepositoryApiClientImpl;
+import com.laserfiche.repository.api.clients.impl.model.Entry;
+import com.laserfiche.repository.api.clients.impl.model.ODataValueContextOfIListOfEntry;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.function.Consumer;
+
+/**
+ *
+ * @author keric
+ */
+public class FileProcessorMain {
+
+     public static void main(String[] args) {
+        String servicePrincipalKey = "z6N-YywpamJ5Dr227qKb";
```

Comment:
Really need to get these keys out of here. Laserfiche wants us to keep these off the internet.

Reply:
I have resolute doubt that there is actually a security risk caused by leaking our credentials to the repo of random text files.  Would you give 150 students the keys to something you actually cared about?